fastlane/lib/fastlane/actions/ensure_xcode_version.rb

```diff
@@ -109,7 +108,7 @@ def self.authors
 
       def self.example_code
         [
-          'ensure_xcode_version(version: "7.2")'
+          'ensure_xcode_version(version: "12.5")'
```

Comment:
we don't support `7.2` anymore and might not be very helpful in Doc example for newbie Fastlane users